src/Type/Row.purs

```diff
@@ -8,8 +8,36 @@ module Type.Row
 import Prim.Row (class Lacks, class Nub, class Cons, class Union)
 import Type.Data.Row (RProxy(..)) as RProxy
 
+-- | Polymorphic Type application
+-- |
+-- | For example...
+-- | ```
+-- | APPLY Maybe Int == Maybe $ Int == Maybe Int
+-- | ```
+type APPLY ∷ ∀ d c . (d → c) → d → c
```

Comment:
Could you use the non-unicode versions: `forall` and `->` here, please? I know it's a bit of a nit but we have a policy for the core libraries that we use ASCII syntax. Also, is there a reason not to use `a` and `b` as the type variable names? I think it would be good if the signature matched that for the term-level version of `$`.

```suggestion
type APPLY ∷ forall a b. (a -> b) -> a -> b
```

I also don't think APPLY and FLIP belong in `Type.Row`, since they don't really have anything to do with rows. Perhaps they could go into a new module `Type.Function`, to mirror the term-level versions which live in `Data.Function`?